docrawl/docrawl_core.py

```diff
@@ -31,13 +31,20 @@
 from selenium.webdriver.support.ui import WebDriverWait
 from selenium.webdriver.support import expected_conditions as EC
 from selenium.webdriver.firefox.service import Service
+from selenium.webdriver.remote.webdriver import WebElement
 
 from webdriver_manager.firefox import GeckoDriverManager
 from webdriver_manager.chrome import ChromeDriverManager
 
-
 from scrapy.selector import Selector
-from keepvariable.keepvariable_core import VarSafe, kept_variables, save_variables, load_variable_safe
+from keepvariable.keepvariable_core import VarSafe, kept_variables, save_variables, load_variable_safe, KeepVariableDummyRedisServer
+
+from typing import Optional
+
+redis: Optional[KeepVariableDummyRedisServer] = None
+
+redis_key_webpage_elements = 'test_user:test_project:test_pipeline:scraping:elements'
+redis_key_screenshot = 'test_user:test_project:test_pipeline:scraping:screenshot'
```

Comment:
project, pipeline, user not anyhow related to docrawl -> this should be different default and overriden where this is applied